pom.xml

```diff
@@ -9,7 +9,7 @@
   <artifactId>hockeyapp</artifactId>
   <name>HockeyApp Plugin</name>
   <description>Uploads iOS, Mac or Android apps to hockeyapp.net</description>
-  <version>1.2.1-SNAPSHOT</version>
+  <version>1.2.2</version>
```

Comment:
Hi, added this since we already use the code (to make it diff from the official release)